ios/linkedin-sdk.framework/Headers/LISDKPermission.h

```diff
@@ -0,0 +1,49 @@
+//
+//  LISDKPermission.h
```

Comment:
Gosh, all this seem to be coming from LinkedIn SDK. This stuff is really badly written..